setup.py

```
@@ -121,7 +121,7 @@ def localversion_func(version: ScmVersion) -> str:
             else "requests>=2.32.2"
         ),
         ("tqdm>=4.66.3,<=4.67.3" if BUILD_TYPE == "release" else "tqdm>=4.66.3"),
-        ("torch>=2.9.0,<=2.10.0" if BUILD_TYPE == "release" else "torch>=2.9.0,<2.11"),
+        ("torch>=2.9.0" if BUILD_TYPE == "release" else "torch>=2.9.0"),
```

Ruff (0.15.9)

[warning] Useless `if`-`else` condition

(RUF034)